README.md

```diff
@@ -36,19 +36,68 @@ scrape_configs:
     metrics_path: /export
     params:
       command: [check_load] # Run the check_load command.
+    relabel_configs:
+      - source_labels: [__address__]
+        target_label: __param_target
+      - source_labels: [__param_target]
+        target_label: instance
+      - target_label: __address__
+        replacement: 127.0.0.1:9275 # Nrpe exporter.
     static_configs:
       - targets: # Targets to run the specified command against.
         - '127.0.0.1:5666'
         - 'example.com:5666'
+
+  - job_name: nrpe_check_apt
+    honor_timestamps: true
+    params:
+      command: [check_apt]
+      ssl: [true]
+    scrape_interval: 3h # no need to poll this more frequently
+    scrape_timeout: 60s # this command could take a long time to execute
+    metrics_path: /export
+    scheme: http
+    follow_redirects: true
     relabel_configs:
       - source_labels: [__address__]
         target_label: __param_target
       - source_labels: [__param_target]
         target_label: instance
       - target_label: __address__
-        replacement: 127.0.0.1:9275 # Nrpe exporter.
+        replacement: nrpe:9275 # Nrpe exporter.
+    static_configs:
+      - targets:
+        - example.com:5666
+
+```
+
+Result Codes in command_status:
+```
+    StatusOK       = 0
+    StatusWarning  = 1
+    StatusCritical = 2
+    StatusUnknown  = 3
 
 ```
+Sample Alert Rule:
```

Comment:
If we're going to have alerts, let's make them standard simple best practice alerts. Using last over time and triggering on a single failure are not best practice.

Reply:
changed as discussed above.